src/bin/cargo/commands/owner.rs

```diff
@@ -6,46 +6,137 @@ use cargo_credential::Secret;
 pub fn cli() -> Command {
     subcommand("owner")
         .about("Manage the owners of a crate on the registry")
-        .arg(Arg::new("crate").value_name("CRATE").action(ArgAction::Set))
+        .arg_quiet()
+        .arg(Arg::new("crate").hide(true))
+        .arg_required_else_help(true)
+        .args_conflicts_with_subcommands(true)
+        .override_usage(
+            "\
+       cargo owner add    <OWNER_NAME> [CRATE_NAME] [OPTIONS]
+       cargo owner remove <OWNER_NAME> [CRATE_NAME] [OPTIONS]
+       cargo owner list   [CRATE_NAME] [OPTIONS]",
```

Comment:
Do we need to update the completions for this?

Reply:
Duplicate of https://github.com/rust-lang/cargo/pull/11879#discussion_r1158576339